### PR TITLE
Ic/fix etcd init

### PR DESCRIPTION
**Pull request by wavesoft**
## High-level description

This commit ensures that etcd initial state file is persisted across cluster reboots. 

## Corresponding DC/OS tickets (required)

  - [COPS-6183](https://jira.d2iq.com/browse/COPS-6183) 2.1 beta - etcd cluster fails and cannot restart if you reboot a master node

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

### CODE REVIEW

**Comment by wavesoft** [comment]
@mesosphere-mergebot request-review @jkoelker @vespian

**Comment by wavesoft** [comment]
@mesosphere-mergebot sync

File: CHANGES.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### Fixed and improved
 
+* * Updated DC/OS UI to [v5.0.41](https://github.com/dcos/dcos-ui/releases/tag/v5.0.41).
```
Comment (jkoelker) [inline CHANGES.md:14]:
```suggestion
```

File: CHANGES.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
...
+
```
Comment (jkoelker) [inline CHANGES.md:15]:
```suggestion
```

**Comment by jkoelker** [comment]
@mesosphere-mergebot backport 2.1

**Pull request merged**